zerver/migrations/0407_rename_emoji_types.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 3.2.13 on 2022-06-28 22:10
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zerver", "0406_alter_realm_message_content_edit_limit_seconds"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="realmuserdefault",
+            name="emojiset",
+            field=models.CharField(
+                choices=[
+                    ("google", "Google"),
+                    ("twitter", "Twitter"),
+                    ("text", "Plain text"),
+                    ("google-blob", "Google blobs"),
+                ],
+                default="google",
+                max_length=20,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="userprofile",
+            name="emojiset",
+            field=models.CharField(
+                choices=[
+                    ("google", "Google"),
+                    ("twitter", "Twitter"),
+                    ("text", "Plain text"),
+                    ("google-blob", "Google blobs"),
+                ],
+                default="google",
+                max_length=20,
+            ),
+        ),
+    ]
```

Comment:
Since this change is just changing the Django metadata about (which doesn't appear in the database), it's better to delete this migration file and instead just reorder the `choices` in the original migration -- saves a migration for everyone, which will be nice for folks testing this PR.

Reply:
Oh neat I didn't know I could do that! Thanks